### ISSUE TITLE
LICENSE bug report

### ISSUE THREAD

**Issue by hyandell** [dib-lab/khmer#1915]
In the LICENSE file, the project states the licensing of the sandbox/memusg gist content to be 'none':

https://github.com/dib-lab/khmer/blob/master/LICENSE#L40

But going to that gist, the licensing is identified as Apache-2.0. It looks like this was a 2015 update, so presumably post when the note was made in the khmer LICENSE file. Just mentioning in case you'd like to update your license file.

**Comment by ctb** [comment]
thank you!!

**Comment by mr-c** [comment]
Yes, that file needs an audit. cqf, rollinghash are completely missing
